charts/rancher-logging/0.1.2/templates/psp.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.global.podSecurityPolicy.enabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ .Release.Name }}-psp
+  labels:
+    app: {{ template "logging.name" . }}
+    chart: {{ template "logging.version" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  allowPrivilegeEscalation: false
+  fsGroup:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  requiredDropCapabilities:
+  - ALL
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  volumes:
+  - configMap
+  - emptyDir
+  - projected
+  - secret
+  - downwardAPI
+  - persistentVolumeClaim
+  - hostPath
+  allowedHostPaths:
+    - pathPrefix: /
```

Comment:
upon further consideration and consultation with @ibuildthecloud this is perhaps too broad. can this be narrowed down at all?

Reply:
The following are the hostPaths mounted by fluentd/log-aggregator. flexVolumeDir & dockerRoot are configurable by helm values.
```
#flexVolumeDir by logaggregator
/usr/libexec/kubernetes/kubelet-plugins/volume/exec
#by fluentd
/var/lib/rancher/fluentd/etc/config/custom
#dockerRoot
/var/lib/docker
/var/log/containers
/var/log/pods
/var/lib/rancher/rke/log
/var/lib/rancher/log-volumes
/var/lib/rancher/fluentd/log
/usr/lib64
```

Reply:
Probably narrow it down to 
/var/lib/rancher
/var/log/containers
/var/log/pods
/usr/lib64
${flexVolumeDir:-"/usr/libexec/kubernetes/kubelet-plugins/volume/exec"}
${dockerRoot:-"/var/lib/docker"}